### PR TITLE
Few more ansible

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

File: shared/templates/static/ansible/aide_build_database.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# disruption = low
+- name: "Build and Test AIDE Database"
+  shell: /usr/sbin/aide --init
+  args:
+    executable: /bin/bash
+  tags:
+    @ANSIBLE_TAGS@
+
+- name: Stage AIDE Database"
+  shell: cp /var/lib/aide/aide.db.new.gz /var/lib/aide/aide.db.gz
```
Comment (mpreisler) [inline shared/templates/static/ansible/aide_build_database.yml]: Use https://docs.ansible.com/ansible/copy_module.html

File: shared/templates/static/ansible/aide_build_database.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: "Build and Test AIDE Database"
+  shell: /usr/sbin/aide --init
+  args:
+    executable: /bin/bash
```
Comment (mpreisler) [inline shared/templates/static/ansible/aide_build_database.yml]: Is this necessary?

Comment (konstruktoid) [inline shared/templates/static/ansible/aide_build_database.yml]: @mpreisler It is, `aide` needs to generate a initial db at start.

Comment (mpreisler) [inline shared/templates/static/ansible/aide_build_database.yml]: Sure but I am talking about `executable: /bin/bash`.

Comment (konstruktoid) [inline shared/templates/static/ansible/aide_build_database.yml]: then no, not necessary.

**Comment by shawndwells** [comment]
Sorry to leave this PR lingering. Heading out on PTO, returning 5june. Will make the adjustments then (unless someone else wants to. PR has 'allow edits from maintainers' checked).

File: shared/templates/static/ansible/aide_build_database.yml (hunk 2 above)
Comment (shawndwells) [inline shared/templates/static/ansible/aide_build_database.yml]: ok, dropped.

File: shared/templates/static/ansible/aide_build_database.yml (hunk 1 above)
Comment (shawndwells) [inline shared/templates/static/ansible/aide_build_database.yml]: done

**Comment by shawndwells** [comment]
Sorry for delays. PR updates made.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**